src/model/job-advert.v1.yaml

```diff
@@ -0,0 +1,45 @@
+$schema: http://json-schema.org/draft-04/schema#
+title: Job advert
+type: object
+allOf:
+  - $ref: ../snippets/job-advert.v1.yaml
+  - properties:
+        description:
+            description: Description
+            type: array
+            items:
+                $ref: ../misc/blocks-all.v1.yaml
+            minItems: 1
+        responsibilities:
+            description: Responsibilities
+            type: array
+            items:
+                $ref: ../misc/blocks-all.v1.yaml
+            minItems: 1
+        experience:
+            description: Experience
+            type: array
+            items:
+                $ref: ../misc/blocks-all.v1.yaml
+            minItems: 1
+        termsAndConditions:
+            $ref: ../blocks/paragraph.v1.yaml
+        offer:
+            description: Experience
+            type: array
+            items:
+                $ref: ../misc/blocks-all.v1.yaml
+            minItems: 1
+        contact:
+            description: Content
```

Comment:
For the email in `Please send your CV and a covering letter explaining your enthusiasm for this position and why you are a great person for this role to hr@elifesciences.org.` which seems pretty important to know when applying for a job